apps/web/src/styles.css

```
@@ -228,6 +274,11 @@ body {
     background: oklch(0.22 0.011 110);
   }
 
+  .changelog-link:hover {
+    color: oklch(0.18 0.01 110);
+    text-decoration-color: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)